magento/pwa-studio
# Whitelist to Allowlist

## Description

This PR updates our `webpack-node-externals` dependency.
This upgrade [changes an input parameter from whitelist to allowlist](https://github.com/liady/webpack-node-externals/releases/tag/v2.0.0). We make the corresponding change in our code as well.

## Related Issue





Closes PWA-961.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. `yarn clean:all && yarn install`
1. Verify that `yarn build` succeeds
1. Verify that `yarn test` succeeds

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
